sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/implementation/models/KeyRotationPolicyAttributes.java

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The key rotation policy attributes.
+ */
+@Fluent
+public final class KeyRotationPolicyAttributes {
+    @JsonProperty(value = "expiryTime")
+    private String expiryTime;
+
+    @JsonProperty(value = "created", access = JsonProperty.Access.WRITE_ONLY)
+    private Long createdOn;
```

Comment:
Does Java have a better type for dates that we can use instead of a `Long`?

Reply:
See [my other comment on this](https://github.com/Azure/azure-sdk-for-java/pull/24452#discussion_r719015249).